### PR TITLE
Release/5.7.0

### PR DESCRIPTION
**Pull request by stefaniacardenas**
### Added
- Public: Added a troubleshooting section to the documentation with details about solving CSP related issues
- UI: Added selfie intro screen
- UI: Option to send cross device secure link using QR code (**Note:** *changes introduced with this UI update include possible breaking changes for integrators with custom translations or copy*)

### Changed
- UI: Unsupported browser message for mobile browsers without getUserMedia API support when `uploadFallback` option is disabled for live document capture and selfie/liveness capture steps
- Internal: Redux and EventEmitter are not in the global scope anymore. The `tearDown` function will only unmount the SDK.

### Fixed
- Internal: Fixed Latest Surge link version not getting updated during release process
- UI: Fixed Liveness capture staying darkened after x-device message dismissed
- Accessibility: Changed Liveness background colour from 66% to 80%

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-895-pr-onfido-sdk-ui-onfido.surge.sh

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +33,7 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 - Internal: Removed `libphonenumber-js` from main bundle. Reduced bundle size limit by 20%.
 - Internal: Use `@sentry/browser` instead of `raven` to track Sentry events
 - UI: New Document Upload screen (**Note:** *changes introduced with this UI update include possible breaking changes for integrators with custom translations or copy*)
+- UI: As part of work to add the QR code option for cross device secure link the UX has been updated for the copy link and SMS options
```
Comment (kopijunkie) [inline CHANGELOG.md]: This should be under Changed section in 5.7.0 (probably my bad from when I was resolving the merge conflict)

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-895-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-895-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-895-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://895-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-895-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-895-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-895-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-895-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-895-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-895-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-895-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://895-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**